cypress/e2e/nodes/Table.spec.js

```diff
@@ -119,6 +119,24 @@ describe('table plugin', () => {
 			.should('have.length', 3)
 			.each(td => cy.wrap(td).should('have.css', 'text-align', 'center'))
 	})
+
+	it('Creates table and add multilines', function() {
+		const tag = 'table'
+		const multilinesContent = 'Line 1\nLine 2\nLine 3'
+
+		cy.getContent().type('Let\'s insert a Table')
+		cy.getActionEntry(`${tag}`).click()
+		cy.getContent()
+			.get('table:nth-of-type(1) tr:nth-child(2) td:nth-child(1)')
```

Comment:
```suggestion
			.find('table:nth-of-type(1) tr:nth-child(2) td:nth-child(1)')
```

should get rid of the eslint warning above:

> It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line